tests/columns/test_interval.py

```diff
@@ -0,0 +1,24 @@
+from tests.testcase import BaseTestCase
+
+
+class IntervalTestCase(BaseTestCase):
+
+    def test_all(self):
+        interval = [
+            ('YEAR', 1),
+            ('MONTH', 2),
+            ('WEEK', 3),
+            ('DAY', 4),
+            ('HOUR', 5),
+            ('MINUTE', 6),
+            ('SECOND', 7)
+        ]
+        columns = ', '.join(['INTERVAL {} {}'.format(v, k)
+                             for k, v in interval])
+        query = 'SELECT {} FROM system.numbers LIMIT 1'.format(columns)
```

Comment:
You can use `'SELECT {}'` as query template. Query `SELECT INTERVAL 1 DAY` works fine without `FROM` and `LIMIT` clauses.